dev/cnf/oss_dependencies.maven

```diff
@@ -31,6 +31,7 @@ com.graphql-java:graphql-java-servlet:6.1.3
 com.graphql-java:graphql-java:11.0
 com.graphql-java:java-dataloader:2.2.1
 com.ibm.db2:jcc:11.1.4.4
+com.ibm.db2:jcc:11.5.0.0
```

Comment:
instead of adding a second DB2 artifact, can we either:
A) re-use the existing one or
B) update the existing one to use 11.5.0.0
so that we only have 1 DB2 driver in the repo